reports/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = '0.1.8'
```

Comment:
Do I need to increment the version to get this deployed to Code Artifact?  No obvious errors in the last code build from previous PR merge.